.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
   "editor.formatOnSave": true,
   "[vue]": {
     "editor.formatOnSave": true,
-    "editor.defaultFormatter": "octref.vetur",
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
   },
   "[typescript]": {
```

Comment:
http://localhost:9000/einstellungen

Events - Eventsverwalten missing padding in card